include/seqan3/range/views/kmer_hash.hpp

```diff
@@ -178,7 +178,7 @@ class kmer_hash_view : public std::ranges::view_interface<kmer_hash_view<urng_t>
     /*!\brief Returns the size of the range, if the underlying range is a std::ranges::sized_range.
      * \returns Size of range.
      */
-    auto size() const
+    auto size()
```

Comment:
```suggestion
    auto size()
    //!\cond
        requires std::ranges::sized_range<urng_t>
    //!\endcond
    {
        using size_type = decltype(std::ranges::size(urange));
        return std::max<size_type>(std::ranges::size(urange) + 1, shape_.size()) - shape_.size();
    }

    auto size() const
    //!\cond
        requires std::ranges::sized_range<urng_t const>
    //!\endcond
    {
        using size_type = decltype(std::ranges::size(urange));
        return std::max<size_type>(std::ranges::size(urange) + 1, shape_.size()) - shape_.size();
    }
```

We should propagate `const` if we can.